admin-console/src/main/webapp/src/components/EmailTemplates/EmailTemplateTable.tsx

```diff
@@ -43,38 +45,22 @@ export const EmailTemplateTable: React.FunctionComponent = () => {
         return undefined;
     }, [ applicationTypesQuery.payload?.status, applicationTypesQuery.payload?.value ]);
 
-    const columns = [ 'Email Template', 'Event Types' ];
-    const rows = [
-        {
-            displayName: 'email template 1',
-            id: '1',
-            eventType: 'event Type 1'
-        },
-        {
-            displayName: 'email template 2',
-            id: '2',
-            eventType: 'event Type 2'
+    const columns = [ 'Instant Email Templates', 'Event Types' ];
 
-        },
-        {
-            displayName: 'email template 3',
-            id: '3',
-            eventType: 'event Type 3'
+    if (templateQuery.loading) {
+        return <Spinner />;
```

Comment:
just interested in it. what does that Spinner do?

Reply:
It spins :grin:
It's a loading animation:
 https://patternfly-react.surge.sh/components/spinner

Reply:
that's unexpected :-D ah. it comes from the patternfl library I never saw before. I actually thought it would be something fancy in html 5 or so.

Reply:
one time I will before a fancy frontend developer fighting for JavaScript in the backend and stuff like that. For sure! :D